MapboxDirections/MBRoute.swift

```diff
@@ -194,6 +197,11 @@ open class Route: NSObject, NSSecureCoding {
      Each route produced by a single call to `Directions.calculate(_:completionHandler:)` has the same route identifier.
      */
     @objc open var routeIdentifier: String?
+    
+    /**
+     The locale to use for spoken instructions.
```

Comment:
Explain that this locale is specifically meant for use with the Mapbox Voice API. Add a note about what `nil` means.